docs/theory-of-operation.md

```diff
@@ -0,0 +1,254 @@
+  If it is `Disabled` then no PXE device should be listed in boot menu and
+  feature `N for PXE boot` in boot menu is also no longer available.
+
+  To `Enable/Disable` booting from Network/PXE press `n` in payload menu.
+
+  Watch [Video](https://asciinema.org/a/240518) showing how to enable/disable
+  PXE boot and what outcome to expect.
+
+
+- **Enable/Disable USB boot**
+
+	This feature let user `Enable/Disable` booting from USB device.   If USB boot
+	is `Enabled` then after restarting apu, USB devices should be listed in `boot
+	menu`.   If USB boot is `Disabled` then after restarting apu, no USB devices
+	should be listed in `boot menu` (even if USB device is attached to USB slot).  
+
+  Watch [Video](https://asciinema.org/a/239796) showing how to enable/disable
+  USB boot.
+
+
+- **Disable serial console and enable with S1 button**
+
+  To `Disable` serial console type `t` in Payload menu. If it is disabled no
+  data will be displayed in serial output.
+
+  After changing state to `Disabled` restoring console can be done only via S1
+  button.
+
+  Watch [video](https://asciinema.org/a/240951) showing how to disable serial
+  console with payload menu option and then enable it with S1 button.
+
+
+- **Redirect console output to COM2**
+
+  Console output is available via COM1 in default. This feature let user
+  redirect console output to COM2.
+  >NOTE: Changing this option from `Disabled` to `Enabled` (or the opposite way)
+  will cause losing current output in terminal. The advice is to open new
+  connection in another terminal window on another COM.
+
+  To `Enable/Disable` redirect console output to COM2 type `k` in payload menu.
+
+  Watch [video](https://asciinema.org/a/240926) showing how to redirect console
+  output to COM2.
+  >NOTE: In above example connection is first opened on 13541 port (COM1). After
+  enabling redirect to COM2, console output is no longer available. New
+  connection is opened on 13542 port (COM2) and console output is available
+  there.
+
+
+- **Enable/Disable CPU boost**
+
+  This feature let user `Enable/Disable` CPU performance boost . To verify if it
+  works, memory test can be done (in boot menu choose `payload[memtest]`).
+
+  To `Enable/Disable` CPU boost type `l` in payload menu.
+
+  Watch [video](https://asciinema.org/a/240928) showing how to enable/disable
+  CPU performance boost.
+  >NOTE: In the example, 2 memory test are performed - one with CPU boost
+  disabled and one with CPU boost enabled. Notice, how long both tests are
+  running and what progress is made during that time. Also look at speed of data
+  transfer to cache memory.
+
+
+- **Enable/Disable UARTC/UARTD**
+	This feature let user enable/disable UARTx.  
+
+	To `Enable/Disable` UARTC type `o` in Payload menu.
+	To `Enable/Disable` UARTD type `p` in Payload menu.
+
+	Watch [Video](https://asciinema.org/a/239817) showing how to enable/disable
```

Comment:
The video does not show that Linux OS detects more serial ports. Please boot Linux OS and show all `ttySx` ports with UARTC/D enabled and disabled to prove it works.